Test Controller/device_test.py

```
@@ -266,625 +86,94 @@ def OnMidiMsg(event, timestamp=0):
     
-    # Try to get actual tempo from the project
+    # 템포 가져오기
     try:
         import mixer
-        tempo = mixer.getCurrentTempo()
-        tempo = tempo/1000
-        print(f"Using project tempo: {tempo} BPM")
-    except (ImportError, AttributeError):
-        print("Using default tempo: 120 BPM")
+        tempo = mixer.getCurrentTempo() / 1000
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)